src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -113,56 +115,202 @@ public SendEmailResponse sendSubmitCaseConfirmationEmail(CaseDetails caseDetails
+                                                      String claimant,
+                                                      String caseNumber,
+                                                      String respondentNames,
+                                                      String hearingDate,
+                                                      String caseId,
+                                                      JSONObject documentJson,
+                                                      ClaimantTse claimantApplication) {
+
+        if (TYPE_C.equals(claimantApplication.getContactApplicationType())
+            || DONT_SEND_COPY.equals(claimantApplication.getCopyToOtherPartyYesOrNo())) {
+            log.info("Acknowledgement email not sent to respondents for this application type");
+            return;
+        }
+        Map<String, Object> respondentParameters = new ConcurrentHashMap<>();
+        addCommonParameters(
+            respondentParameters,
+            claimant,
+            respondentNames,
+            caseId,
+            caseNumber
+        );
+        respondentParameters.put(
+            HEARING_DATE,
+            hearingDate
+        );
+        respondentParameters.put("shortText", SHORT_TEXT_MAP.get(claimantApplication.getContactApplicationType()));
+        respondentParameters.put("datePlus7", LocalDate.now().plusDays(7).toString());
+
+        String emailToRespondentTemplate;
+        if (Stream.of(typeB).anyMatch(appType -> Objects.equals(
+            appType,
+            claimantApplication.getContactApplicationType()
+        ))) {
+            emailToRespondentTemplate = notificationsProperties.getRespondentTseEmailTypeBTemplateId();
+        } else {
+            emailToRespondentTemplate = notificationsProperties.getRespondentTseEmailTypeATemplateId();
+        }
+        respondentParameters.put("linkToDocument", Objects.requireNonNullElse(documentJson, ""));
+
+        caseData.getRespondentCollection()
+            .forEach(resp -> {
+                String respondentEmailAddress = NotificationsHelper.getEmailAddressForRespondent(
+                    caseData,
+                    resp.getValue()
+                );
+                if (isNullOrEmpty(respondentEmailAddress)) {
+                    log.info("Respondent did not have an email address associated with their account");
+                } else {
+                    try {
+                        notificationClient.sendEmail(
+                            emailToRespondentTemplate,
+                            caseData.getClaimantType().getClaimantEmailAddress(),
+                            respondentParameters,
+                            caseId
+                        );
+                        log.info("Sent email to respondent");
+                    } catch (NotificationClientException ne) {
+                        throw new NotificationException(ne);
+                    }
```

Comment:
Could we put it in separate method sentEmailToRespondent()?

Reply:
Only similarity is that its wrapped in a try catch. Each time I call sendEmail I pass different parameters. You have made me spot that I was sending this email to the claimant email address rather than respondent. I've fixed that.